lib/solution_tester.rb

```diff
@@ -110,6 +110,6 @@ def run_tests_for_stage(stage)
   end
 
   def solution_exists_for_stage?(stage)
-    File.directory?("#{solutions_dir}/#{@language.slug}/#{stage.slug}")
+    File.directory?(@course.stage_path(@language, stage))
```

Comment:
```suggestion
    File.directory?(stage.solution_dir)
```

(i.e. no backward compatibility)